### PR TITLE
Fortifier42 - Meta Updates

### PR DESCRIPTION
**Pull request by Fortifier42**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -881,7 +953,17 @@ public void registerCoreMembers() {
...
         // @Usage
-        // TODO: Document Command Details
+        // Use to set a player's experience to 0
+        // - experience 0
+        // @Usage
+        // Use give give a player 1 level
+        // - experience give level 1
+        // @Usage
+        // Use to give a player with the name Morphan1 10 experience points.
+        // - experience give 10 player<p@Morphan1>
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: ???

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -881,7 +953,17 @@ public void registerCoreMembers() {
...
+        // @Usage
+        // Use to take 1 level from a specific player
+        // - experience take level 1 player:<player>
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: ????

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -802,11 +868,19 @@ public void registerCoreMembers() {
...
+        // It allows or blocks to be equipped in an entity's armor/hand slots, this inclueds horses.
         // @Tags
         // <e@entity.equipment>
         // @Usage
-        // TODO: Document Command Details
+        // Use to equip a stone block on a player's head
+        // - equip <player> head:stone
+        // @Usage
+        // Use to equip a diamond sword into the hand of an npc named bob
+        // - equip <n@bob> hand:diamond_sword
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: ???

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 4]
```diff
@@ -693,12 +747,24 @@ public void registerCoreMembers() {
...
+        // - displayitem stone <player.location>
+        // @Usage
+        // Use to display a diamond sword dropped at 12,64,-847 in world survival
+        // - displayitem diamond_sword 12,64,-847,survival
+        // @Usage
+        // Use to display redstone dust dropped at -54,100,-87 in world creative disappear after 10 seconds
+        // - displayitem redstone -54,100,-87,creative duration:10s
+        // @Usage
+        // Use to save the dropped item to save entry 'item_dropped'
+        // - displayitem redstone -54,100,-87,creative duration:10s save:item_dropped
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: `i@`, `l@`, ...

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 5]
```diff
@@ -333,11 +354,28 @@ public void registerCoreMembers() {
...
+        // Use to make a npc break a block in an interact script.
+        // - break <location>
+
+        // @Usage
+        // Use to make a npc with the id 12 break a block at 17,64,-87
+        // - break 17,64,-87 entity:n@12
+
+        // @Usage
+        // Use to make a npc with the name bob break a block at 17,64,-87 and start digging from 5 blocks away
+        // - break 17,64,-87 entity:n@bob radius:5
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: `l@`

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 6]
```diff
@@ -896,14 +978,30 @@ public void registerCoreMembers() {
...
+        // Use to create an explosion at a player's location
+        // - explode <player.location>
+
+        // @Usage
+        // Use to create an explosion with a power radius of 3 at a NPC's location
+        // - explode power:3 <npc.location>
+
+        // @Usage
+        // Use to create an explosion with a power radius of 3 at a 12,12,-1297 in a world called survival which breaks blocks
+        // - explode power:3 12,12,-1297,survival breakblocks
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: `l@`

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 7]
```diff
@@ -470,9 +510,13 @@ public void registerCoreMembers() {
         // @Author spaceemotion, mcmonkey
         // @Group world
         // @Description
-        // TODO: Document Command Details
+        // This command forces a chunk to load and keeps the chunk loaded until it is removed,
+        // the server shuts down or the duration expires. All normal events such as redstone,
+        // NPC actions, mob spawning and weather effects will continue to take place while
+        // the chunk is loaded.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Have you tested this claim?

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 8]
```diff
@@ -333,11 +354,28 @@ public void registerCoreMembers() {
...
+        // Use to make a npc break a block in an interact script.
+        // - break <location>
+
+        // @Usage
+        // Use to make a npc with the id 12 break a block at 17,64,-87
+        // - break l@17,64,-87 entity:n@12
+
+        // @Usage
+        // Use to make a npc with the name bob break a block at 17,64,-87 and start digging from 5 blocks away
+        // - break l@17,64,-87 entity:n@bob radius:5
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: all the above ^ worlds

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 9]
```diff
@@ -333,11 +354,28 @@ public void registerCoreMembers() {
...
+
         // @Tags
-        // TODO: Document Command Details
+        // <n@npc.navigator.is_navigating>
+        // <n@npc.navigator.target_location>
+
         // @Usage
-        // TODO: Document Command Details
+        // Use to make a npc break a block in an interact script.
+        // - break <location>
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: wtf is `<location>`?

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 7 above)
Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Which one?

Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: That everything keeps happening as normal if you just chunkload it

**Pull request merged**